testdata/bless_provider/terraform/accounts/foo/fogg.tf

```diff
@@ -1,12 +1,20 @@
 # Auto-generated by fogg. Do not edit
 # Make improvements in fogg, so that everyone can benefit.
 
+
```

Comment:
why do we get all these blank lines now?

Reply:
Terraform fmt seems to add way more blank lines. Don't know why, even though the open issue says it won't add new lines.

https://github.com/hashicorp/terraform/issues/21360#issuecomment-493807349

Reply:
I think that comment is mislead, it seems that they only care about horizontal spacing.